packages/elements/src/components/link-with-icon/link-with-icon-story.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
Should we follow the same directory structure where stories go under a `__stories__` folder?